nashville-software-school/yak-25
# registration form now sends to api and session storage

## Description of Proposed Changes

- Registration form now appears when clicked and upon submission, data is sent to API and session storage

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout kb-registration

cd api
json-server -p 8088 -w db.database

open new terminal
cd to root
npm start

```

1. Click "Register New Account"
1. Fill out form
1. Check Session Storage - data should be there
1. Check API  - data should be there
